astropy/modeling/fitting.py

```diff
@@ -1117,7 +1117,7 @@ def __init__(self, calc_uncertainties=False, use_min_max_bounds=True):
         self._use_min_max_bounds = use_min_max_bounds
         super().__init__()
 
-    def objective_function(self, fps, *args):
+    def objective_function(self, fps, *args, **context):
```

Comment:
Any reason not to just list the supported kwargs explicitly?

Reply:
done